src/primer_filter.hpp

```diff
@@ -36,21 +43,27 @@ struct Primer {
     vector<size_t> mapped_nodes_ids;
 };
 
+/**
+ * Primer_pair struct contains primer pair attributesm including left primer, right primer,
+ * linear product size, minimum and maximum product size on the sequence graph, and boolean on
+ * whether the primers locate in low variation region of the sequence graph.
+ */
 struct Primer_pair {
     Primer left_primer;
     Primer right_primer;
     size_t linear_product_size;
     size_t min_product_size;
     size_t max_product_size;
+    bool no_variation = false;
 };
 
 class Primer_finder {
```

Comment:
Just a style thing, but it should be PrimerFinder and PrimerPair, or primer_pair. In vg we usually use camel case for classes.